exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Transaction.kt

```diff
@@ -189,6 +198,10 @@ open class Transaction(private val transactionImpl: TransactionInterface) : User
         executedStatements.clear()
     }
 
+    internal fun getRetryInterval(): Long = if (repetitionAttempts > 0) {
+        maxOf((maxRepetitionDelay - minRepetitionDelay) / (repetitionAttempts + 1), 1)
+    } else 0
+
```

Comment:
This has been extracted to an internal function in `Transaction` class because it's used across 2 files: Suspended.kt and ThreadLocalTransactionManager.kt.

I move it instead as a `Transaction` extension outside of the class, if that's preferred.